src/coreclr/tests/src/Directory.Build.props

```diff
@@ -102,7 +102,7 @@
     </When>
     <Otherwise>
       <PropertyGroup>
-        <ConfigurationErrorMsg>$(ConfigurationErrorMsg);Unknown OSGroup [$(OSGroup)] specificed in your project.</ConfigurationErrorMsg>
+        <ConfigurationErrorMsg>$(ConfigurationErrorMsg);Unknown BuildOS [$(BuildOS)] specificed in your project.</ConfigurationErrorMsg>
```

Comment:
This property is never used. I would remove the whole branch and add validation to the ps1/sh script.